tests/modules/layers/test_normalizations.py

```diff
@@ -61,3 +62,63 @@ def test_rms_norm_core_algo():
     assert_expected(output_ones, input_ones)
     assert_expected(output_fixed, fixed_expected, atol=1e-04, rtol=1e-05)
     assert output_fixed.dtype == torch.float32
+
+
+def test_simple_rmsnorm():
+    dims = 12
+    srms_norm = SimpleRMSNorm(dims)
+
+    input_bf16_ones = torch.ones(dims, dtype=torch.bfloat16)
+
+    input_fixed_fp32 = torch.tensor(
+        [
+            0.999,
+            1.1111,
+            2.222,
+            3.333,
+            4.444,
+            5.555,
+            6.678,
+            7.987,
+            8.123,
+            9.101010,
+            110.00,
+            120.2589,
+        ],
+        dtype=torch.float32,
+    )
+
+    expected_output_bf16_ones = torch.tensor(
+        [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0],
+        dtype=torch.bfloat16,
+    )
+    expected_output_fixed = torch.tensor(
```

Comment:
curious, how do we get these expected outputs? If we get them by running the function itself, I'm not sure how that provides confidence on the correctness? cc @ebsmothers

Reply:
This is a valid point, but note that for previous test cases where I did this (effectively compute something in two different ways where the end result should be the same) got pushback that doing that was 'weird' and should just use constant inputs and expected outputs. 

Regardless - In this case I've established it works as expected via a model training run (i.e actual use testing), and from there locked in the expected outputs using two known inputs.

Reply:
Sorry, just getting to this now. In this case I would say that we can actually hand compute the expected output for some simple inputs. Not sure that can be done super easily with the existing `input_fixed_fp32` but for something like e.g. [3.0, 4.0, 3.0, 4.0, 3.0, 4.0, 3.0, 4.0] it'd be nice and easy. Tbh not a big deal either way